companies/fraunhofer-de.json

```diff
@@ -0,0 +1,90 @@
+        "Fraunhofer-Institut für Mikroelektronische Schaltungen und Systeme IMS",
+        "Fraunhofer-Institut für Mikrostruktur von Werkstoffen und Systemen IMWS",
+        "Fraunhofer-Einrichtung für Mikrosysteme und Festkörper-Technologien EMFT",
+        "Fraunhofer-Institut für Mikrotechnik und Mikrosysteme IMM",
+        "Fraunhofer-Institut für Molekularbiologie und Angewandte Oekologie IME",
+        "Fraunhofer-Institut für Nachrichtentechnik, Heinrich-Hertz-Institut HHI",
+        "Fraunhofer-Institut für Naturwissenschaftlich-Technische Trendanalysen INT",
+        "Fraunhofer-Institut für Offene Kommunikationssysteme FOKUS",
+        "Fraunhofer-Institut für Optronik, Systemtechnik und Bildauswertung IOSB",
+        "Fraunhofer-Institut für Organische Elektronik, Elektronenstrahl- und Plasmatechnik FEP",
+        "Fraunhofer-Institut für Photonische Mikrosysteme IPMS",
+        "Fraunhofer-Institut für Physikalische Messtechnik IPM",
+        "Fraunhofer-Institut für Produktionsanlagen und Konstruktionstechnik IPK",
+        "Fraunhofer-Institut für Produktionstechnik und Automatisierung IPA",
+        "Fraunhofer-Institut für Produktionstechnologie IPT",
+        "Fraunhofer-Informationszentrum Raum und Bau IRB",
+        "Fraunhofer-Institut für Schicht- und Oberflächentechnik IST",
+        "Fraunhofer-Institut für Sichere Informationstechnologie SIT",
+        "Fraunhofer-Institut für Silicatforschung ISC",
+        "Fraunhofer-Institut für Siliziumtechnologie ISIT",
+        "Fraunhofer-Institut für Software- und Systemtechnik ISST",
+        "Fraunhofer-Institut für Solare Energiesysteme ISE",
+        "Fraunhofer-Institut für System- und Innovationsforschung ISI",
+        "Fraunhofer-Institut für Techno- und Wirtschaftsmathematik ITWM",
+        "Fraunhofer-Institut für Toxikologie und Experimentelle Medizin ITEM",
+        "Fraunhofer-Institut für Umwelt-, Sicherheits- und Energietechnik UMSICHT",
+        "Fraunhofer-Institut für Verfahrenstechnik und Verpackung IVV",
+        "Fraunhofer-Institut für Verkehrs- und Infrastruktursysteme IVI",
+        "Fraunhofer-Institut für Werkstoff- und Strahltechnik IWS",
+        "Fraunhofer-Institut für Werkstoffmechanik IWM",
+        "Fraunhofer-Institut für Werkzeugmaschinen und Umformtechnik IWU",
+        "Fraunhofer-Institut für Windenergiesysteme IWES",
+        "Fraunhofer-Institut für Zelltherapie und Immunologie IZI",
+        "Fraunhofer-Institut für Zerstörungsfreie Prüfverfahren IZFP",
+        "Fraunhofer-Institut für Zuverlässigkeit und Mikrointegration IZM"
+    ],
+    "address": "Hansastraße 27 c\n80686 München",
+    "phone": "+49 89 12050",
+    "fax": "+49 89 12057531 ",
+    "email": "datenschutz@zv.fraunhofer.de",
+    "web": "https://www.fraunhofer.de",
+    "sources": [
+        "https://www.fraunhofer.de/de/impressum.html",
+        "https://www.fraunhofer.de/de/datenschutzerklaerung.html"
+    ]
+}
```

Comment:
As we have a `datenschutz@` email, we can set a `suggested-transport-medium` of `email`.